pkg/database/vercode.go

```diff
@@ -195,6 +195,16 @@ func (db *Database) FindVerificationCodeByUUID(uuid string) (*VerificationCode,
 	return &vc, nil
 }
 
+// ExpireCode saves a verification code as expired.
+func (db *Database) ExpireCode(uuid string) (time.Time, error) {
+	now := time.Now().UTC()
+	vc := VerificationCode{
+		ExpiresAt:     now,
+		LongExpiresAt: now,
+	}
+	return now, db.db.Model(&vc).Where("uuid = ?", uuid).Debug().Update(&vc).Error
```

Comment:
same comment I put on the other review.

Needs to check that it makes sense to actually invalidate the code
* hasn't been claimed
* hasn't already expired